### ISSUE TITLE
New locations

### ISSUE THREAD

**Issue by oliveiraxton** [tommyinb/PokemonGoMap#2]
Hello bro, i have interested to expand the map to my region. I fork the project. Need helps to modify

**Comment by tommyinb** [comment]
It is great to hear that. Let's branch and modify the project for your region.

**Comment by bluelatios** [comment]
i'll cheer you all :dancer:

**Comment by josephtsai1126** [comment]
Hope this great tool can expand to Taipei or Taiwan. Thank you.

**Comment by zakyn** [comment]
Hi

What is needed for modification? I just briefly looked through, is it pulling from Skiplagged or?

**Comment by tommyinb** [comment]
Yes, it is pulling data from Skiplagged every 3 minutes.

1) DownloadData.cs line 19: change bound in URL to suit your area
2) bin/index.html line 234: change to the center location of your area
3) bin/Debug/Account.xml: create an XML file for Account.cs to let program commit results itself

Thank you for your contribution. Thank you.

**Comment by zakyn** [comment]
Thanks for the heads up. Will try take a look over the weekend when I have time.
Pretty Cool!

**Comment by zakyn** [comment]
I took a look at skiplagged data for Singapore, seems it doesn't fully scan all areas in Singapore . Not sure how accurate or useful this would be as the data will only be concentrated on the areas they do scan

**Comment by tommyinb** [comment]
Yes. Some area are scanned with lower frequency while some are with higher frequency. Many believe that it bases on population density.
If Skiplagged misses an area for more than 10 minutes, a handful of pokemons may be undiscovered. But, I think the chance of total ignorance for 10 minutes is small.
Also, as a report is generated by 24 hours data, a small loss should not be significant. From my experience in Hong Kong, the map is fairly accurate.

**Comment by zakyn** [comment]
Actually it's not just the area, they seem to only scan 4 spots in Singapore, and completely miss out the west part of Singapore and various other portions. 

I don't know if they do cover it though I have to say I never really monitor it for over 24 hours. I will try look at it again next week as only my office comp has .NET.

Good stuff all around though and I love the Hong Kong map!
